pkg/process/v5/writer.go

```diff
@@ -138,3 +173,15 @@ func normalizeSeverity(metadata *grypeDB.VulnerabilityMetadata, reader grypeDB.V
 	}
 	metadata.Severity = newSeverity
 }
+
+func (p ProviderMetadata) Write(path string) error {
+	providerMetadataJSON, err := json.MarshalIndent(p, "", "  ")
+	if err != nil {
+		return fmt.Errorf("unable to marshal provider metadata: %w", err)
+	}
+	//nolint:gosec
+	if err = os.WriteFile(path, providerMetadataJSON, 0644); err != nil {
```

Comment:
is there a specific need to be permissive + lint ignore?
```suggestion
	if err = os.WriteFile(path, providerMetadataJSON, 0600); err != nil {
```

Reply:
Just to ensure enterprise can read the file without munging permissions.